core/scss/components/main-nav/_main-nav.scss

```diff
@@ -0,0 +1,318 @@
+
+      > li {
+        > a {
+          @include fancy-hover;
+          @include padding(null 0);
+          @include margin(0 1.16em 0 0);
+          font-size: 1.05em;
+          border-bottom: 0;
+
+          &:active {
+            font-weight: $font-bold;
+          }
+        }
+
+        &:last-child > a {
+          @include margin(null 0 null null);
+        }
+
+        &.su-main-nav__item--current > a {
+          font-weight: $font-bold;
+        }
+      }
+    }
+  }
+}
+
+.su-main-nav__toggle {
+  @include fancy-hover;
+  @include padding(0 0 1em);
+  @include margin(0);
+  display: flex;
+  flex-direction: column-reverse;
+  width: 40px;
+  background-color: transparent;
+  color: $color-black;
+  font-size: .8em;
+  font-weight: $font-normal;
+  line-height: .7;
+
+  &::after {
+    @include margin(0 auto);
+    display: inline-block;
+    width: 30px;
+    height: 26px;
+    background: url("#{$image-path}/hamburger-black.svg") no-repeat 3px 0;
+    content: "";
+  }
+
+  &[aria-expanded="true"] {
+    &::before {
+      background-color: $color-black;
+    }
+
+    &::after {
+      width: 26px;
+      background: url("#{$image-path}/close-black.svg") no-repeat 3px 0;
+      background-size: 20px 20px;
+    }
+  }
+
+  &:hover,
+  &:active,
+  &:focus {
+    background-color: transparent;
+    color: $color-black;
+    box-shadow: none;
+  }
+
+  @include grid-media('lg') {
+    display: none;
+  }
+
+  @include grid-media-max('md') {
+    &[aria-expanded="false"] + .su-main-nav__menu-lv1 {
+      display: none;
+    }
+
+    &[aria-expanded="true"] + .su-main-nav__menu-lv1 {
+      position: absolute;
+    }
+  }
+
+  @include grid-media-only('md') {
+    &[aria-expanded="true"]  + .su-main-nav__menu-lv1 {
+      max-width: 21em;
+    }
+  }
+}
+
+.su-main-nav__menu-lv1 > li {
+  @include grid-media('lg') {
+    > a {
+      color: $color-black;
+      transition: color .3s ease-out;
+
+      &:hover,
+      &:focus {
+        color: $color-dark-red;
+      }
+
+      &:active,
+      &[aria-expanded="true"] {
+        color: $color-black;
+      }
+
+      &[aria-expanded="true"]:hover {
+        color: $color-dark-red;
+      }
+    }
+  }
+
+  &.su-main-nav__item--current {
+    > a {
+      &::before {
+        transform: scaleY(1);
+        background-color: #ec0513;
+      }
+
+      &:hover,
+      &:focus {
+        &::before {
+          left: 6px;
+          background-color: #ec0513;
+        }
+      }
+    }
+
+    @include grid-media('lg') {
+      > a {
+        &::before {
+          background-color: $color-black;
+        }
+
+        &:hover,
+        &:focus {
+          &::before {
+            left: 0;
+            background-color: $color-dark-red;
+          }
+        }
+      }
+    }
+  }
+}
+
+.su-main-nav__menu-lv2 {
+  @include padding(null null null 1em);
+
+  @include grid-media('lg') {
+    @include padding(1px null null .6em);
+    @include margin(null null null -18px);
```

Comment:
If the first item in the top level nav has a subnav, the -18px margin causes that first subnav to extend outside the nav's box. If the main nav is ever styled to be full bleed, these means the focus and current indicators aren't visible.

Reply:
I just pushed a fix for the styleguide fullscreen mode. I think it would be really rare that it will take up the full width of the browser when it's actually used on a site and if that happens, the developer can add their own custom code?

Reply:
Resolved for now - might revisit later.